.github/workflows/e2e-matrix.yaml

```diff
@@ -31,7 +31,10 @@ jobs:
       - name: Harden Runner
         uses: step-security/harden-runner@5c7944e73c4c2a096b17a9cb74d65b6c2bbafbde # v2.9.1
         with:
-          egress-policy: audit
+          disable-telemetry: true
+          disable-sudo: true
+          egress-policy: block
+          allowed-endpoints: >
```

Comment:
Does this mean no allowed endpoints, and it by default inherits the allowed endpoints from any parent workflow, or does this explicitly call out to inherits the endpoints?

Reply:
Assuming its doing something, but if not, can we remove?

Reply:
Maybe just drop a comment to explain?

Reply:
This means no allowed endpoints (besides some built-in ones, like GH cache) for this job (and no inheritance). Will add a comment.